LICENSE

```diff
@@ -1,14 +1,202 @@
-   Copyright 2012 R.I.Pienaar
```

Comment:
See below: https://github.com/puppetlabs/puppetlabs-concat/pull/378/files#diff-aa14599dc3fdfa9cc81a7c3c247b77b9

This change is to better align with ASF's licence application guidelines.